internal/jet/select_lock.go

```diff
@@ -37,6 +44,23 @@ func (s *selectLockImpl) SKIP_LOCKED() RowLock {
 func (s *selectLockImpl) serialize(statement StatementType, out *SQLBuilder, options ...SerializeOption) {
 	out.WriteString(s.lockStrength)
 
+	if len(s.of) > 0 {
+		out.WriteString("OF")
+
+		for i, of := range s.of {
+			if i > 0 {
+				out.WriteString(", ")
+			}
+
+			table := of.Alias()
+			if table == "" {
+				table = of.TableName()
+			}
+
+			out.WriteIdentifier(table, false)
```

Comment:
You'll need to remove `alwaysQuote` completely to avoid the quote. It is an optional parameter.

Reply:
Done in https://github.com/go-jet/jet/pull/288/commits/92a923b98be3182bcaa2ecd8d229985cf76baf93